flower/command.py

```diff
@@ -8,140 +8,138 @@
 
 from logging import NullHandler
 
+import click
 from tornado.options import options
 from tornado.options import parse_command_line, parse_config_file
 from tornado.log import enable_pretty_logging
-from celery.bin.base import Command
+from celery.bin.base import CeleryCommand
 
 from . import __version__
 from .app import Flower
 from .urls import settings
 from .utils import abs_path, prepend_url
 from .options import DEFAULT_CONFIG_FILE, default_options
 
-
 logger = logging.getLogger(__name__)
+ENV_VAR_PREFIX = 'FLOWER_'
 
 
-class FlowerCommand(Command):
-    ENV_VAR_PREFIX = 'FLOWER_'
+@click.command(cls=CeleryCommand,
+               context_settings={
+                   'ignore_unknown_options': True
+               })
+@click.argument("torando_argv", nargs=-1, type=click.UNPROCESSED)
```

Comment:
Why not just use click options?

Reply:
I wanted to keep it consistent with the previous approach which delegates tornado's options to `tornado.options.parse_command_line`.
otherwise, there would have to be a mapping between tornado-options to click-options, defined on this command, which I think is hard to maintain for future tornado releases. 
(it can be possible if tornado would have exposed its options in a way that can be transformed to click.option, but I don't have experience with tornado's API)

Reply:
s/torando/tornado/

Reply:
thanks